src/itest/java/uk/gov/companieshouse/pscstatementdataapi/config/AbstractIntegrationTest.java

```diff
@@ -22,9 +19,6 @@ public abstract class AbstractIntegrationTest extends AbstractMongoConfig {
     @MockBean
     public PscStatementApiService pscStatementApiService;
 
-    @MockBean
```

Comment:
Is there a reason to change this from a MockBean to a mock in the steps? Just because if we ever introduce a new steps file with a metrics call we'd have to mock in 2 places now